packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -85,6 +85,8 @@ export interface IContainerRuntimeMetadata extends ICreateContainerMetadata {
     readonly gcFeature?: GCVersion;
     /** Counter of the last summary happened, increments every time we summarize */
     readonly summaryCount?: number;
+    /** The session timeout for all clients */
```

Comment:
Please add a little more detail here. Like if this is present, the session for this container will expiry after this time and the container will be closed.